src/test/java/core/basesyntax/service/handlers/OperationStrategyImplTest.java

```diff
@@ -0,0 +1,55 @@
+package core.basesyntax.service.handlers;
+
+import core.basesyntax.model.Transaction;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class OperationStrategyImplTest {
+    private static OperationStrategyImpl operationStrategyImpl = new OperationStrategyImpl();
+
+    @BeforeClass
+    public static void beforeClass() {
+        operationStrategyImpl.getOperationHandlersMap().put(Transaction.Operation.BALANCE,
+                new BalanceOperationHandler());
+        operationStrategyImpl.getOperationHandlersMap().put(Transaction.Operation.SUPPLY,
+                new SupplyOperationHandler());
+        operationStrategyImpl.getOperationHandlersMap().put(Transaction.Operation.PURCHASE,
+                new PurchaseOperationHandler());
+        operationStrategyImpl.getOperationHandlersMap().put(Transaction.Operation.RETURN,
+                new ReturnOperationHandler());
+    }
+
+    @Test
+    public void getBalanceOperation_Ok() {
+        Class<BalanceOperationHandler> expected = BalanceOperationHandler.class;
+        OperationHandler operationHandler = operationStrategyImpl
+                .get(Transaction.Operation.BALANCE);
+        Class<BalanceOperationHandler> actual = (Class<BalanceOperationHandler>)
+                                                    operationHandler.getClass();
+    }
```

Comment:
I think we should compare expected and actual result somehow (same in other tests)

Reply:
ahahaha, well, what a nice test it was without a comparison. Thank you!